great_expectations_provider/example_dags/example_great_expectations_dag.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import annotations
+from datetime import datetime
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+import pandas as pd
+from airflow import DAG
+from airflow.models.baseoperator import chain
+from airflow.operators.empty import EmptyOperator
+from great_expectations_provider import (
+    GXValidateDataFrameOperator,
+    GXValidateBatchOperator,
+    GXValidateCheckpointOperator,
+)
+from airflow.decorators import task
+from airflow.models.param import Param
+from great_expectations import ExpectationSuite, ValidationDefinition, Checkpoint
```

Comment:
Where are these ExpectationSuite, ValidationDefinition and Checkpoint interfaces defined? Or are they yet to be defined?

Reply:
these are great expectations objects, available in the top level GX namespace